### ISSUE TITLE
os.access required argument error checking if pinned cert is readable

### ISSUE THREAD

**Issue by chrisrink10** [elastic/apm-agent-python#433]
**Describe the bug**: 

I just started trying to deploying the certificate pinning feature released in 4.2.0 (#405) which I was super excited about (!). However, I found that there is a bug in [the validation for the supplied CA bundle](https://github.com/elastic/apm-agent-python/blob/master/elasticapm/conf/__init__.py#L175). Specifically, [`os.access`](https://docs.python.org/3.6/library/os.html#os.access) requires a `mode` argument (and it appears to have required this argument all the way back to [2.7](https://docs.python.org/2.7/library/os.html#os.access)), but one is not supplied.

**To Reproduce**

1. Start Python service (mine is Django) instrumented with Elastic APM and specify a custom CA certificate via `SERVER_CERT` config.
2. See error:

```
Traceback (most recent call last):
  File "/usr/local/bin/celery", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/celery/__main__.py", line 16, in main
    _main()
  File "/usr/local/lib/python3.6/site-packages/celery/bin/celery.py", line 322, in main
    cmd.execute_from_commandline(argv)
  File "/usr/local/lib/python3.6/site-packages/celery/bin/celery.py", line 496, in execute_from_commandline
    super(CeleryCommand, self).execute_from_commandline(argv)))
  File "/usr/local/lib/python3.6/site-packages/celery/bin/base.py", line 294, in execute_from_commandline
    return self.handle_argv(self.prog_name, argv[1:])
  File "/usr/local/lib/python3.6/site-packages/celery/bin/celery.py", line 488, in handle_argv
    return self.execute(command, argv)
  File "/usr/local/lib/python3.6/site-packages/celery/bin/celery.py", line 420, in execute
    ).run_from_argv(self.prog_name, argv[1:], command=argv[0])
  File "/usr/local/lib/python3.6/site-packages/celery/bin/worker.py", line 223, in run_from_argv
    return self(*args, **options)
  File "/usr/local/lib/python3.6/site-packages/celery/bin/base.py", line 248, in __call__
    ret = self.run(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/celery/bin/worker.py", line 257, in run
    **kwargs)
  File "/usr/local/lib/python3.6/site-packages/celery/worker/worker.py", line 96, in __init__
    self.app.loader.init_worker()
  File "/usr/local/lib/python3.6/site-packages/celery/loaders/base.py", line 114, in init_worker
    self.import_default_modules()
  File "/usr/local/lib/python3.6/site-packages/celery/loaders/base.py", line 108, in import_default_modules
    raise response
  File "/usr/local/lib/python3.6/site-packages/celery/utils/dispatch/signal.py", line 288, in send
    response = receiver(signal=self, sender=sender, **named)
  File "/usr/local/lib/python3.6/site-packages/celery/fixups/django.py", line 84, in on_import_modules
    self.worker_fixup.validate_models()
  File "/usr/local/lib/python3.6/site-packages/celery/fixups/django.py", line 122, in validate_models
    self.django_setup()
  File "/usr/local/lib/python3.6/site-packages/celery/fixups/django.py", line 118, in django_setup
    django.setup()
  File "/usr/local/lib/python3.6/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python3.6/site-packages/django/apps/registry.py", line 120, in populate
    app_config.ready()
  File "/usr/local/lib/python3.6/site-packages/elasticapm/contrib/django/apps.py", line 60, in ready
    self.client = get_client()
  File "/usr/local/lib/python3.6/site-packages/elasticapm/contrib/django/client.py", line 73, in get_client
    instance = client_class()
  File "/usr/local/lib/python3.6/site-packages/elasticapm/contrib/django/client.py", line 89, in __init__
    super(DjangoClient, self).__init__(config, **inline)
  File "/usr/local/lib/python3.6/site-packages/elasticapm/base.py", line 98, in __init__
    self.config = Config(config, inline_dict=inline)
  File "/usr/local/lib/python3.6/site-packages/elasticapm/conf/__init__.py", line 208, in __init__
    setattr(self, field, new_value)
  File "/usr/local/lib/python3.6/site-packages/elasticapm/conf/__init__.py", line 66, in __set__
    value = self._validate(instance, value)
  File "/usr/local/lib/python3.6/site-packages/elasticapm/conf/__init__.py", line 78, in _validate
    value = validator(value, self.dict_key)
  File "/usr/local/lib/python3.6/site-packages/elasticapm/conf/__init__.py", line 175, in __call__
    elif not os.access(value):
TypeError: Required argument 'mode' (pos 2) not found
```

**Expected behavior**: App starts up.

**Environment (please complete the following information)**
- OS: Amazon Linux (using Docker image `python:3.6-slim-stretch`)
- Python version: 3.6.x
- Framework and version: Django 2.1.7
- APM Server version: n/a
- Agent version: 4.2.0

**Additional context**

- Agent config options (these env vars are set into the corresponding settings in a Django `settings.py`)
  <details>

    <summary>Click to expand</summary>

    ```
      AZ_ELASTIC_APM_SERVER_URL:    https://apm-server:8200
      AZ_ELASTIC_APM_SERVER_CA:     /etc/ssl/certs/staging.ca.pem
      AZ_ELASTIC_APM_SECRET_TOKEN:  ...
      AZ_ELASTIC_APM_ENVIRONMENT:   staging
      AZ_ELASTIC_APM_SERVICE_NAME:  ...
    ```
  </details>
- `requirements.txt`:
  <details>
    <summary>Click to expand</summary>

    ```
    n/a
    ```
  </details>

**Comment by beniwohli** [comment]
Well that's embarrassing  🙈

Thanks for reporting, and great to hear that you like the feature (once it works). I'll release a version with a fix ASAP!

**Comment by chrisrink10** [comment]
Thank you!

**Comment by beniwohli** [comment]
@chrisrink10 I just released version 4.2.1 of the agent which contains a fix for this issue. Thanks again for the heads up!
